src/libstd/io/cursor.rs

```diff
@@ -31,6 +31,7 @@ use slice;
 /// over `T` itself. Instead, specific implementations are provided for various
 /// in-memory buffer types like `Vec<u8>` and `&[u8]`.
 #[stable(feature = "rust1", since = "1.0.0")]
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
```

Comment:
Can this leave out `Copy` for now? I've generally tried to avoid that wherever possible. I also think that it's fine to remove `Eq`/`PartialEq` as they should be pretty easy to add back in the future, and I think that comparing I/O objects for equality is a pretty rare operation.